projects/step-frontend/src/lib/modules/_common/components/sidebar/sidebar.component.html

```diff
@@ -0,0 +1,63 @@
+<div id="sidebar-logo-and-burgermenu">
+  <span class="visible-when-collapsed"
+    ><label for="collapsable-sidebar-toggle-checkbox"><i class="glyphicon glyphicon-menu-hamburger"></i></label
+  ></span>
+  <span class="sidebar-aligned-text"><img class="sidebar-logo" alt="" [src]="logo" /></span>
+</div>
+<div id="sidebar-home-and-settings">
+  <span class="visible-when-collapsed sidebar-home-button" (click)="navigateTo('')"
+    ><i class="glyphicon glyphicon-home"></i
+  ></span>
+  <span class="sidebar-home-and-settings-divider">|</span>
+  <span (click)="navigateTo('admin')" *ngIf="_authService.hasRight('admin')" class="sidebar-home-and-settings-icon"
+    ><i class="glyphicon glyphicon-cog"></i
+  ></span>
+</div>
+<div id="sidebar-categories">
+  <div class="sidebar-categories-categories">
+    <span
+      (click)="navigateTo('admin')"
+      *ngIf="_authService.hasRight('admin')"
+      class="only-visible-when-collapsed sidebar-home-and-settings-icon-collapsed"
+      style="font-size: medium"
+      ><i class="glyphicon glyphicon-cog"></i
+    ></span>
+    <span>Categories</span>
+  </div>
+  <div id="sidebar-categories-tabs">
+    <!-- MENU -->
+    <div
+      class="sidebar-collapsable-tab"
+      *ngFor="let mme of _viewRegistryService.getCustomMainMenuEntries(); index as i"
+      [attr.lastSidebarMainMenuLabel]="i === mainMenuEntriesAmount"
+    >
+      <input #mainMenuCheckBox name="{{ mme.label }}" id="collapsable-tab-{{ mme.label }}" type="checkbox" />
```

Comment:
Just for code style, you should not use curly brackets to bind values to components. Since it is a string in this case it works the same but the correct way would be: [name]="mme.label"

Reply:
Also I have a strong dislike against abrviations, so instead of "mme" I would use either "mainMenuEntry" to be specific or "entry" to be unspecific.

Reply:
I agree absolutely again, me using something like `mme` is an exception, usually I got problems the other way round - with too long variable names :) . Variable names should be self explanatory.